packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -69,6 +70,17 @@ InputDateTimePicker.propTypes = {
 	useSeconds: PropTypes.bool,
 	useUTC: PropTypes.bool,
 	timezone: PropTypes.string,
+	defaultTimeValue: PropTypes.oneOfType([
+		PropTypes.shape({
+			hours: PropTypes.string,
+			minutes: PropTypes.string,
+			seconds: PropTypes.string,
+		}),
+		PropTypes.shape({
+			hours: PropTypes.string,
+			minutes: PropTypes.string,
+		}),
+	]),
```

Comment:
You don't need to redefine that to set seconds as optional. By default, a prop is optional, unless you set `.isRequired`.
Define `defaultTimeValue ` it once as non required. And set the hours and minutes as required